backend/daemondriver.cpp

```diff
@@ -0,0 +1,175 @@
+// Copyright 2016 Husky Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <thread>
+#include <unordered_map>
+#include <vector>
+
+#include "boost/thread.hpp"
+
+#include "backend/daemondriver.hpp"
+#include "backend/workerdriver.hpp"
+
+#include "core/config.hpp"
```

Comment:
I think so. It will seem more clear using library from using submodule. Thank you.